src/07-projects/project-items.apib

```diff
@@ -71,4 +71,16 @@ Get a list of project items.
                 + updated_at: `2018-12-05T16:44:33+00:00` (string)
         + meta (object)
             + page (Pagination)
-            + matches: `120` (number)
\ No newline at end of file
+            + matches: `120` (number)
+
+### projectItems.updateCost [POST /projectItems.updateCost]
```

Comment:
Q: Why not `.update`?

Reply:
A: Because we only allow to update the cost.

Reply:
If that field is the only thing that makes sense updating for a project item, in general, I think it's still an option. Will there be an update endpoint in the future?

Reply:
Updating the cost is quite specific as it also needs to update the purchase price of the originating order item, as a side-effect.